switch_mod/project/dispatch.py

```diff
@@ -160,6 +160,22 @@ def init_dispatch_timepoints(m):
     mod.PROJ_DISPATCH_POINTS = Set(
         dimen=2,
         initialize=init_dispatch_timepoints)
+    # make set array showing all timepoints when each project is active
```

Comment:
Move comments like this to the docstring at the top of define_components, and possibly to the doc attribute of the component itself. Ex:
```python
    mod.PROJ_ACTIVE_TIMEPOINTS = Set(mod.PROJECTS, initialize=proj_active_timepoints_rule,
    	doc=("The set of timepoints when a given project may be active. " +
    	     "This will be an empty set if a project is retired before the " +
    	     "start of the study."))
```